tests/e2e/config/global-teardown.js

```diff
@@ -23,13 +22,12 @@ module.exports = async ( config ) => {
 
 	// Clean up the consumer keys
 	const keysRetries = 5;
-	for ( let i = 0; i < keysRetries; i++ ) {
+	for ( let i = 1; i <= keysRetries; i++ ) {
```

Comment:
Is there a reason there's an additional wrapper of retries around the default in the `user.login()` function? 🤔 
I assume it's for redundancy -- or maybe for the error logging? -- but why not just use the `retries` parameter on the `user.login()` function?

Reply:
I moved the login step to outside of the retries loop. Now the retries are just for actually generating the keys and we assume we're already logged in.